chart/templates/scanner/deployment.yaml

```diff
@@ -0,0 +1,123 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- with (mergeOverwrite (deepCopy .Values.global.deploymentAnnotations) .Values.scanner.deploymentAnnotations) }}
+  annotations:
+    {{- range $key, $value := . }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+  {{- end }}
+  name: {{ include "kyoo.scanner.fullname" . }}
+  labels:
+    {{- include "kyoo.labels" (dict "context" . "component" .Values.scanner.name "name" .Values.scanner.name) | nindent 4 }}
+spec:
+  replicas: {{ .Values.scanner.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "kyoo.selectorLabels" (dict "context" . "name" .Values.scanner.name) | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        {{- with (mergeOverwrite (deepCopy .Values.global.podAnnotations) .Values.scanner.podAnnotations) }}
+        {{- range $key, $value := . }}
+        {{ $key }}: {{ $value | quote }}
+        {{- end }}
+        {{- end }}
+      labels:
+        {{- include "kyoo.labels" (dict "context" . "component" .Values.scanner.name "name" .Values.scanner.name) | nindent 8 }}
+        {{- with (mergeOverwrite (deepCopy .Values.global.podLabels) .Values.scanner.podLabels) }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.scanner.imagePullSecrets | default .Values.global.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.global.securityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "kyoo.scanner.serviceAccountName" . }}
+      containers:
+        - name: main
+          image: {{ .Values.scanner.kyoo_scanner.image.repository | default (printf "%s/kyoo_scanner" .Values.global.image.repositoryBase) }}:{{ default (include "kyoo.defaultTag" .) .Values.scanner.kyoo_scanner.image.tag }}
+          imagePullPolicy: {{ default .Values.global.image.imagePullPolicy }}
+          args:
+            {{- with .Values.scanner.kyoo_scanner.extraArgs }}
+              {{- toYaml . | nindent 12 }}
+            {{- end }}
+          env:
+            - name: SCANNER_LIBRARY_ROOT
```

Comment:
You are missing the `LIBRARY_IGNORE_PATTERN` env var

Reply:
I tried not to add every single configuration on the first pass.  The power user can set individual environmental variables via `scanner.kyoo_scanner.extraEnv`.  

Alternatively we can expose general settings via kyoo.languages or kyoo.unloggedPermissions.  Definitely would need some help picking out the most important ones to showcase.

Reply:
I think it would be best to expose everything, I try to keep configuration options to a minimum.

It can be a challenge to find which service will read the option, making it harder to set up.

Reply:
Still mapping out what user inputs should look like.

Would prefer to have `SCANNER_LIBRARY_ROOT` be calculated from media.volumeMounts[0].mountPath[0] value.  For `LIBRARY_IGNORE_PATTERN` I would likely create variable inside of `media.ignorePattern`

Reply:
Added setting for configuring `LIBRARY_IGNORE_PATTERN` . closing